docs/firmware/payload_uboot.md

```diff
@@ -0,0 +1,32 @@
+U-Boot as a payload to OpenSBI
+==============================
+
+U-Boot is an open-source primary boot loader. It can be used as first and/or
+second stage boot loader in an embedded environment. Here, it is used as next
+stage bootloader to OpenSBI if given as a payload to OpenSBI.
```

Comment:
In the context of OpenSBI, U-boot can be specified as a payload to OpenSBI firmware, becoming a the boot stage following OpenSBI firmware execution.